firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -52,6 +52,15 @@ final class MicrosurveyTests: BaseTestCase {
 
         XCTAssertTrue(app.images[AccessibilityIdentifiers.Microsurvey.Survey.firefoxLogo].exists)
         XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Microsurvey.Survey.closeButton].exists)
+        let tablesQuery = app.scrollViews.otherElements.tables
+        let firstOption = tablesQuery.cells.firstMatch
+        firstOption.tap()
+        XCTAssertEqual(firstOption.label, "Very satisfied")
```

Comment:
```suggestion
        mozWaitForElementToExist(firstOption)
        XCTAssertEqual(firstOption.label, "Very satisfied")
        mozWaitForValueToContains(firstOption, "1 out of 6")
```

Reply:
I am getting errors with `mozWaitForValueToContains` cannot be found

Reply:
It works if I use `mozWaitForValueContains(firstOption, value: "1 out of 6")`